src/_articles/index.md

```diff
@@ -1,8 +1,6 @@
 ---
-layout: default
 title: Articles
-description: "Read about the Dart language and tools with this collection of articles, style guides, and more."
-permalink: /articles
+description: Read early articles about the Dart language and libraries.
```

Comment:
If we delete or move every file under /articles, then yes.

Reply:
Oh, didn't realize that this was the only way of getting to them

Reply:
I don't actually link to this page or most of the articles, iirc, but if people find an article, they're likely to walk up the URL chain. We don't have to list articles we don't want people to visit, but I'd like to have a page to avoid unnecessary 404s.